lib/utils/AbstractQueryConfiguration.dart

```diff
@@ -30,12 +32,15 @@ abstract class AbstractQueryConfiguration {
   String? cc;
   List<ProductField>? fields;
 
+  List<Parameter> additionalParameters;
+
   AbstractQueryConfiguration({
     this.language,
     this.languages,
     this.lc,
     this.cc,
     this.fields,
+    this.additionalParameters = const [],
```

Comment:
I am not sure since the analyzer doesn't complain but don't we have to specify the list type before the `[]`

Reply:
Hmmmm I never do and never had any problems with the `const []` declarations so far.
Do you want me to put a type here?

Reply:
As long as the analyzer does not complain, let's say it's OK.
I've just double-checked and both syntaxes are accepted (`const []` and `const <Parameter>[]`).
@M123-dev Probably `dart` knowing that it's a `const` and that the field is a `List<Parameter>` understands that it's implicitly a `const <Parameter>[]`. I don't understand either in which specific cases the analyzer actually complains.